apps/todo-list-client/src/app/generic-button/generic-button.component.html

```diff
@@ -0,0 +1,3 @@
+<button type="button" class="m-2 rounded-xl bg-white p-4 text-center font-bold">
```

Comment:
add de event to the button please, like (event)='changeAction()'

Reply:
I'm done with this, thank you